client/package-lock.json

```diff
@@ -14796,8 +14796,7 @@
             "ansi-regex": {
               "version": "2.1.1",
               "bundled": true,
-              "dev": true,
-              "optional": true
+              "dev": true
```

Comment:
The changes in `client/package-lock.json` are *not* related to the Cypress upgrade and improvement

Reply:
Yep. It's due to `docker-compose up`

Reply:
But you can *not* included as part of the commit. Just a minor suggestion for future PRs.